### ISSUE TITLE
Personal hashtags for readers

### ISSUE THREAD

**Issue by KolyaKorruptis** [mastodon/mastodon#20429]
### Pitch

As a reader I would like to create collections of posts that in my mind belong together, but do not share a common tag by their creators. These collections would only be visible and accessible to me.

### Motivation

Tagging is a lot of work and requires knowledge about previously existing similar content/tags. The main benefactor of hashtags are readers, not creators. Therefore tagging is often done sloppily, idiosyncratically or not at all. 

Giving readers the ability to publicly tag other people's content would open them up for malicious tagging. Private tags prevent this problem while still allowing readers to group content as they see fit.

**Comment by veale** [comment]
I (think) this could be done elegantly with little extra effort by allowing hashtags, which can be followed in 4.0.0, to also be added to lists.

**Comment by trwnh** [comment]
Duplicate of #1208

**Comment by KolyaKorruptis** [comment]
@trwnh Not the same thing. I suggested that the **reader** can create tags, to group content by multiple other creators. 

#1208 suggests that a creator can subcategorize his own content: Alice#Cats

**Comment by trwnh** [comment]
So are you suggesting a list of multiple hashtags? I think that can be done in the advanced web UI, you can create a column with multiple hashtags iirc

**Comment by KolyaKorruptis** [comment]
@trwnh Yes, you can. But those tags are added by the content creators. I am talking about tags that a reader can add to other people's content to create collections that suit this reader.

**Comment by KolyaKorruptis** [comment]
Alice makes a post about elephants without any tags.
Bob reads this post and adds an #elephant tag that only works for Bob.
Now when Bob checks #elephant then Alice's post is there, but only for him.
